Assets/Scripts/Audio/AudioGroup.cs

```diff
@@ -0,0 +1,87 @@
+﻿using CollectionExtensions;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "AudioGroup", menuName = "Audio/New Audio Group")]
+public class AudioGroup : ScriptableObject
```

Comment:
When using this system in practice, we are supposed to call EmitSound(someAudioGroup) in a script, I assume? And have someAudioGroup as a member of the component we call EmitSound() from?

Reply:
Yes! Although the design is negotiable!
The thought there behind not having the AudioGroup as a member of AudioEmitter is the eventuality that you might want a system to play audioclips from multiple different groups, and yet from the same audio source.

That can also be done in other ways if we see that that's more feasible - if for example; we see that the majority of cases have a 1<=>1 relation between AudioEmitter and AudioGroup.

Leaving it like that for now... the method signature and "who" holds the AudioGroup reference is a fairly simple refactor if we see it would make more sense.